javascript/javascript1/week3/uniquefeature.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>
+            notes saving app with extra feature
+        </title>
+        <meta charset="utf-8">               
+    </head>
+    <body>
+        <script>
+          "use strict"
+console.log("Savenote app with my unique feature.")
+/*with in savenote app, i want add check mark which will help to separate finished and unfinished tasks*/ 
+const notes = [];
+const finishedTasks=[];
+const unfinishedTasks=[];
+function saveNote(content, id,check) {  
+   notes.push({content:content, id:id});
+
+  if (check ===true){
+      finishedTasks.push({content:content, id:id});    
+    
+  }else if (check === false|| check === undefined){
```

Comment:
else is enough you don't need other condition

Reply:
i removed else condition..